tests/pkg-manifest-url/OPS/package.opf

```diff
@@ -12,8 +12,8 @@
     <dc:title>Content in arbitrary directory</dc:title>
```

Comment:
Since this is no longer an RS test, the description on line 8, and on page 1 of the publication, should be updated to describe content behavior (I guess referencing EPUBCheck?).

Reply:
(Copying @mattgarrish here, because that is a sligthly more general editorial issue and also @rdeltour because all this originates in https://github.com/w3c/epub-specs/issues/1918. I believe that came up in another thread.) 

The problem I see is that there is no tight separation between the RS and the Core spec in this case. Issue https://github.com/w3c/epub-specs/issues/1918 came up through the wish to avoid unnecessary redundancy in terms of normative statements between the two specs. But that also means that, for an RS, there are some general requirements about properly interpreting the core content where the details might be normatively stated in the content/core spec rather than specific to the RS. I believe this is unavoidable. This is one of those: the issue of spreading the XHTML file in an arbitrary folder is of course a content format issue (hence epubcheck's) _but_ also an RS issue insofar as the RS must process those references and not rely on some fixed pattern.

From a standard point of view, in my view this is correctly stated in the introductory statement of [§3](https://www.w3.org/TR/epub-rs-33/#sec-pkg-doc-manifest):

> Reading Systems MUST process the [Package Document](https://www.w3.org/TR/epub-33/#sec-package-doc)

And we should try to keep at that. The statements in the RS spell out those parts/details that are relevant for the RS only, but the assumption is that the normative statements in the Core spec are also "honored" by the RS whenever appropriate.

For our testing environment: I would propose to refer to this statement for all the tests in this category. I am sure there will be more cases, including for other similar statements (look at all the yellow lines...)